src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingHome.java

```diff
@@ -196,6 +238,122 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
             // 'Capture' as many citizens into this house as possible
             addHomelessCitizens();
         }
+
+        if (childCreationTimer > childCreationInterval)
+        {
+            childCreationTimer = 0;
+
+            // Spawn a child when adults are present and the house has space, so level 3+
+            if (femalePresent && malePresent && colony.getCitizenManager().getCurrentCitizenCount() < colony.getCitizenManager().getMaxCitizens())
+            {
+                CitizenData mom = null;
+                CitizenData dad = null;
+
+                for (CitizenData data : getAssignedCitizen())
+                {
+                    if (data.isFemale() && !data.isChild())
+                    {
+                        mom = data;
+                    }
+                    else if (!data.isChild())
+                    {
+                        dad = data;
+                    }
+
+                    if (mom != null && dad != null)
+                    {
+                        break;
+                    }
+                }
+
+                if (mom == null || dad == null)
+                {
+                    return;
+                }
+
+                Log.getLogger().info("Spawning new citizen");
+
+                CitizenData newCitizen = colony.getCitizenManager().createAndRegisterNewCitizenData();
+
+                Random rand = new Random();
+
+                // Inheriting stats from parents + some randomness. Capped by happiness
+                int str = (mom.getStrength() + dad.getStrength()) / 2 + rand.nextInt(3) - rand.nextInt(3);
+                str = str > colony.getOverallHappiness() ? (int) colony.getOverallHappiness() : str;
+
+                int cha = (mom.getCharisma() + dad.getCharisma()) / 2 + rand.nextInt(3) - rand.nextInt(3);
+                cha = cha > colony.getOverallHappiness() ? (int) colony.getOverallHappiness() : cha;
+
+                int dex = (mom.getDexterity() + dad.getDexterity()) / 2 + rand.nextInt(3) - rand.nextInt(3);
+                dex = dex > colony.getOverallHappiness() ? (int) colony.getOverallHappiness() : dex;
+
+                int end = (mom.getEndurance() + dad.getEndurance()) / 2 + rand.nextInt(3) - rand.nextInt(3);
+                end = end > colony.getOverallHappiness() ? (int) colony.getOverallHappiness() : end;
+
+                int intelligence = (mom.getIntelligence() + dad.getIntelligence()) / 2 + rand.nextInt(3) - rand.nextInt(3);
+                intelligence = intelligence > colony.getOverallHappiness() ? (int) colony.getOverallHappiness() : intelligence;
+
+                newCitizen.setIsChild(true);
+                newCitizen.setStrength(str);
```

Comment:
can we give the kid a certain gender by chance? Like something: If 60% of the colony are male, then the chance is 60% to become female (to get to a good average case a lot of citizens of one gender die)

Reply:
I do not think that is a good idea.
This should just be 50-50. If a lot of your men die, that is the players problem. These kinds of problems exist in the real world as well.

Reply:
ye dont think either we really need a set gender balance